actors/user/src/main/java/org/sunbird/user/actors/UserManagementActor.java

```diff
@@ -679,6 +679,16 @@ private void setStateValidation(
   private Map<String, Boolean> updatedUserFlagsMap(
       Map<String, Object> userMap, Map<String, Object> userDbRecord) {
     Map<String, Boolean> userBooleanMap = new HashMap<>();
+    if(userDbRecord.get(JsonKey.EMAIL) != null && userDbRecord.get(JsonKey.EMAIL_VERIFIED) == null) {
```

Comment:
need to handle scenario for not null and value as true as well. same for below code

Reply:
yes taken care for existing users with true value.